src/util/EventDefinitionsUtil.js

```diff
@@ -0,0 +1,53 @@
+import { filter } from 'min-dash';
+
+/**
+ *
+ * @param {ModdleElement} element
+ * @param {String} [type] - optional
```

Comment:
According to JSdoc should be `{string}` afaik

Reply:
+1